articles/introduction-to-deep-learning/index.md

```diff
@@ -0,0 +1,123 @@
+
+3. Healthcare
+
+Deep Learning is completely revolutionizing the healthcare and the medical industries. According to [Medium](https://medium.com/breathe-publication/top-15-deep-learning-applications-that-will-rule-the-world-in-2018-and-beyond-7c6130c43b01), AI has enabled healthcare and medical industries to advance tremendously.
+Clinical researchers use DL to find a cure for untreatable diseases. DL helps with a speedy diagnostic of dangerous conditions. Many cancer tests, such as the Pap test and Mammograms, use DL to examine cells image using a microscope.
+
+4. Virtual Assistants
+
+Siri and Google Assistants are approved deep learning virtual assistants. Deep Learning enables virtual assistants to learn and understand commands given by a user. Virtual assistants then execute by providing the appropriate answer naturally. Virtual assistants use Deep Learning to learn about you, from what you search most.
+
+5. Fraud Detection
+
+Banking and financial sectors are benefiting from Deep Learning to detect transaction fraud. [Autoencoders](https://www.mygreatlearning.com/blog/deep-learning-applications/) in Tensorflow are being used to catch credit card fraud, thus saving a lot of money from fraudsters. Fraud prevention is done based on recognizing patterns in customer transactions.
+
+6. Image Recognition
+
+Image Recognition using Deep Learning aims to recognize and learn content in images. Also, Deep Learning aims to understand their surrounding. Image Recognition is mostly used in gaming and social media.
+
+7. Entertainment
+
+Entertainment industries such as [Netflix](https://www.netflix.com/ke-en/) recommends to their viewers what they need. Deep Learning enables the entertainment industry to understand the consumer's behavior. This helps to provide an exciting experience to clients. Deep Learning is revolutionizing the filmmaking process. Cameras can learn body language and conduct voice synthesis. Deep Learning also helps to emulate someone's voice to imbibe in virtual characters.
+
+### Working of Neural Networks
+Neural Networks are algorithms modeled to function like the human brain. Neural Networks is a system of hardware and software patterned after neurons operation in the human mind. Neurons are decision-makers.
+
+![Neural Networks](/engineering-education/introduction-to-deep-learning/neural-networks.jpg)<br>
+[\*image source\*](https://medium.com/coinmonks/the-artificial-neural-networks-handbook-part-1-f9ceb0e376b4)
+
+Neural Networks comprise of the input layer, hidden layers, and the output layer.
+
+The input layer is where the neural network acquires data in numeral values between 0 and 1. Any data can be represented as a number, whether it is words, sound, or image. Each input neuron represents a single feature.
```

Comment:
The numbers can be anything. If the data is normalized, then the numeral values will be between 0 and 1. Talk about feature extractors such as SIFT, SURF, word vectors to emphasize on the fact that data can be represented as *vectors*.